README.md

```diff
@@ -23,6 +23,7 @@
 
 | status | `jupyterlite-pyodide-kernel` | `jupyterlite-core` |  `jupyterlab`  |   `notebook`   |  `retrolab`  |
 | :----: | :--------------------------: | :----------------: | :------------: | :------------: | :----------: |
+|  pre   |           `0.4.*`            |    `>=0.4,<0.5`    | `>=4.2.0,<4.3` | `>=7.2.0,<7.3` |      -       |
```

Comment:
maybe take this opportunity to add the pyodide/python versions to the matrix?

Reply:
Done in https://github.com/jupyterlite/pyodide-kernel/pull/106/commits/cf7d410031adf862099b4d44b94f95e694f49339